news/2021-02-01-mappers-report-january.md

```diff
@@ -0,0 +1,133 @@
+
+- **osu!:** [Okoratu](https://osu.ppy.sh/users/1623405)
+- **osu!taiko:** [Greenshell](https://osu.ppy.sh/users/8693851)
+- **osu!catch:** [JierYagtama](https://osu.ppy.sh/users/7483452) & [-Joni-](https://osu.ppy.sh/users/9988837)
+- **osu!mania:** [Rivals_7](https://osu.ppy.sh/users/4610379) & [Voxa](https://osu.ppy.sh/users/9229539)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open.
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+## <a id="mappers-showcase"></a>Mappers’ Showcase
+
+As we had all modes covered with the last issues of the Mappers’ Report it’s time again for the Storyboarder showcase! The unofficial 5th game mode of osu! gets way to less attention. We have many amazing storyboarders out there who are able to improve your playing experience by a lot. So let us jump right in!
+Thanks to [Walavouchey](https://osu.ppy.sh/users/5773079) for this month’s picks!
+
+### VenerableNyanta
+ 
+With increasingly refined storyboards this year, [VenerableNyanta](https://osu.ppy.sh/users/12243368) has shown their determination for improvement project after project! It's easy to tell they have an eye for design when the visuals fill the screen with nothing but pure eye candy!
+ 
+As a brilliant first ranked storyboard, [ClariS - Kimi no Shiranai Monogatari (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1191205#osu/2633220) contains a few sections with varying ideas fitting the bright music! Here you can find a blend of going for visuals on every single note, lots of stars and sparkles and immersive backgrounds, all pulled together with tasteful transitions.
+
+What about slower, more... moody songs? VenerableNyanta has the right idea when going for [Josh A & Jake Hill - Worst Mistakes (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1312306#osu/2719818), showing the contrast between a more neon-coloured intense section and the black and white quieter parts. The design work demonstrates an ever-improving understanding of what makes a picture look neat by establishing a memorably unique identity.
+
+The crazy visuals sure seem fitting in [HiTECH NINJA - Modelista](https://osu.ppy.sh/beatmapsets/1148442#osu/2397737), capturing the energy of the song with more neon colours, rapid movement, exceptional design, and a good bit of humour as well! If you decide to check out this graveyarded gem, as you absolutely should, you'll surely notice how VenerableNyanta must've had a good chuckle when creating it! As tedious and daunting of a process storyboarding might seem, there are people who do know how to have fun while doing so.
+
+### TheDuckMask
+
+Confident storyboarder for a couple of years now, [TheDuckMask](https://osu.ppy.sh/users/7405768) has shown expertise in nowadays more niche sides of storyboard design.
```

Comment:
```suggestion
Confident storyboarder for a couple of years now, [TheDuckMask](https://osu.ppy.sh/users/7405768) has shown their expertise in what is nowadays considered more niche sides of storyboard design.
```

Reply:
Technically not needed, and I personally prefer the conciseness